docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -1279,6 +1279,153 @@ Sending Access-Request of id 74 to 127.0.0.1 port 18120
 rad_recv: Access-Accept packet from host 127.0.0.1:18120, id=74, length=20
 ----
 
+Portal Modules
+~~~~~~~~~~~~~~
+
+The PacketFence captive portal flow is highly customizable. This section will cover the 'Portal Modules' which are used to define the behavior of the captive portal.
+
+NOTE: When upgrading from a version that doesn't have the portal modules, the PacketFence Portal Modules configuration already comes with defaults that will fit most cases and that offer the same behavior as previous PacketFence version. Meaning, all the available Portal Profile sources are used for authentication, then the available provisioners will be used.
+
+First, a brief description of the available Portal Modules : 
```

Comment:
cut the space before the : - in English there is none. Do that everywhere please.